lib/style.css

```diff
@@ -461,3 +461,29 @@ button.goog-buttonset-default:hover {
 .material-detail-view .top-tracks {
   background-color: <<BACK_PRIMARY>>;
 }
+
+paper-button#unsubscribe-playlist-button .playlist-subscribed,
+paper-button#unsubscribe-playlist-button:hover .playlist-unsubscribe {
+    color: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:unsubscribe"] path:last-child
+{
+    fill: <<FORE_PRIMARY>>;
+}
+
+paper-button#unsubscribe-playlist-button iron-icon[icon="sj:subscribed"] path
```

Comment:
Ah yeah, good spot. I tackled them both separately because they both have different numbers of child elements. Woops. 2 secs.